pkgs/tools/filesystems/irods/common.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, bzip2, zlib, autoconf, automake, cmake, gnumake, help2man , texinfo, libtool , cppzmq , libarchive, avro-cpp, boost, jansson, zeromq, openssl, pam, libiodbc, kerberos, gcc, libcxx, cross, which }:
+
+# Common attributes of irods packages
+
+with stdenv;
+
+# Boost and avro-c++ have to be built with the same stdenv
```

Comment:
Someone to help me finalize this package, or merge it as is? My code is maybe not perfect, but I'm ready to learn... if someone can at least point me to the part of the doc that could help me... or, please, merge ;-)